webviz_config/templates/radixconfig.yaml.jinja2

```diff
@@ -0,0 +1,66 @@
+apiVersion: radix.equinor.com/v1
+kind: RadixApplication
+metadata:
+  name: {{ application_name }}
+spec:
+  environments:
+    - name: prod
+      build:
+        from: main
+  components:
+    - name: auth
+      # TODO: https://github.com/oauth2-proxy/oauth2-proxy/blob/master/CHANGELOG.md#breaking-changes-5
+      image: quay.io/pusher/oauth2_proxy:v6.1.1 # 4.1.0 # see https://pusher.github.io/oauth2_proxy/configuration
+      ports:
+        - name: http
+          port: 8000
+      publicPort: http
+      secrets:
+        - OAUTH2_PROXY_CLIENT_SECRET
+        - OAUTH2_PROXY_COOKIE_SECRET
+        - OAUTH2_PROXY_REDIRECT_URL  # Redirect URL a "secret" so it can be configured per cluster, but it's not sensitive information.
+      environmentConfig:
+        - environment: prod
+          variables:
+            OAUTH2_PROXY_SCOPE: openid offline_access profile {{ app_registration_id }}/user_impersonation email
+            OAUTH2_PROXY_CLIENT_ID: {{ app_registration_id }}
+            OAUTH2_PROXY_COOKIE_REFRESH: "60m" # How often should the token be refreshed. Default for Azure AD is currently 60m
+            OAUTH2_PROXY_EMAIL_DOMAINS: "*" # Any email address registered in AD should be accepted
+            OAUTH2_PROXY_HTTP_ADDRESS: "http://:8000" # The port oauth2_proxy listens on
+            OAUTH2_PROXY_OIDC_ISSUER_URL: "https://login.microsoftonline.com/{{ tenant_id }}/v2.0"
+            OAUTH2_PROXY_PASS_ACCESS_TOKEN: "true" # Pass the access token upstream (to the proxy component)
+            OAUTH2_PROXY_PASS_BASIC_AUTH: "false" # Disable unused default
+            OAUTH2_PROXY_PASS_USER_HEADERS: "false" # Disable unused default
+            OAUTH2_PROXY_PROVIDER: "oidc" # The "azure" provider doesn't seem to work properly
+            OAUTH2_PROXY_REDIS_CONNECTION_URL: "redis://auth-state:6379" # Where to store session info (the auth-state component)
+            OAUTH2_PROXY_SESSION_STORE_TYPE: "redis" # We're using Redis for storing session info instead of cookies (cookies would get too big)
+            OAUTH2_PROXY_SKIP_PROVIDER_BUTTON: "true" # We don't want a "click to login" page; just issue a redirect
+            OAUTH2_PROXY_UPSTREAMS: "http://main:5000" # Where authenticated requests are routed to: the web application
+
+    - name: auth-state
+      image: redis:5-alpine
+      ports:
+        - name: redis
+          port: 6379
+
+    - name: main
+      src: "."
+      ports:
+        - name: http
+          port: 5000
+      environmentConfig:
+        - environment: prod
+          resources:
+            requests:
+              memory: "400Mi"
+              cpu: "100m"
+            limits:
+              memory: "4G"
+              cpu: "1000m"
```

Comment:
Is it ideal to default these settings? 
I guess we can override them after initial setup, but we've seen significantly higher memory usage on localhost than what has been defaulted here?
Also: I would think that such a spread between requested and maximum memory usage could lead to out of memory errors if you need the limit value, and you're a bit unlucky?

Reply:
I think it is outside scope in this PR to intelligently find good values based on what the user has added to the app. But we should review the default values before merge :+1: https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/ and https://www.radix.equinor.com/docs/reference-radix-config/#resources give some information.